docs/user-manual/en/amqp.md

```diff
@@ -155,4 +155,20 @@ This contains a real example for configuring amqpIdleTimeout:
 
 ```xml
 <acceptor name="amqp">tcp://0.0.0.0:5672?amqpIdleTimeout=0;tcpSendBufferSize=1048576;tcpReceiveBufferSize=1048576;protocols=AMQP;useEpoll=true;amqpCredits=1000;amqpMinCredits=300;directDeliver=false;batchDelay=10</acceptor>
-```
\ No newline at end of file
+```
+
+## Web Sockets
+
+Apache ActiveMQ Artemis also supports AMQP over [Web
+Sockets](https://html.spec.whatwg.org/multipage/web-sockets.html).  Modern web
+browsers which support Web Sockets can send and receive AMQP messages.
+
+MQTT over Web Sockets is supported via a normal AMQP acceptor:
```

Comment:
Think you mean AMQP here, MQTT over the AMQP accepter seems wrong

Reply:
Fixed. Thanks!